qiskit/algorithms/amplitude_amplifiers/grover.py

```diff
@@ -29,6 +29,13 @@
 class Grover(AmplitudeAmplifier):
     r"""Grover's Search algorithm.
 
+        .. note::
+
+            If you want to learn more about Grover's Search algorithm, check out the
+            `Qiskit Textbook <https://qiskit.org/textbook/ch-algorithms/grover.html>`_.
+            or the `Qiskit Tutorials
+            <https://qiskit.org/documentation/tutorials/algorithms/07_grover_examples.html>`_.
```

Comment:
The textbook link, to me, is more if you are interested in the theory and how to build out such an algorithm yourself, whereas the latter is more instructional around using this specific implementation here. I don't know if its worth qualifying the text in the this regard. Linking them with `or` above might imply equivalence at some level, but they are kinda different.  One can be interested in both aspects of course,

Reply:
Like so?
```suggestion
            If you want to learn more about the theory behind Grover's Search algorithm, check 
            out the `Qiskit Textbook <https://qiskit.org/textbook/ch-algorithms/grover.html>`_.
            or the `Qiskit Tutorials
            <https://qiskit.org/documentation/tutorials/algorithms/07_grover_examples.html>`_
            for more concrete how-to examples.
```

Reply:
sounds good